### ISSUE TITLE
servant-response fails to build w/ GHC 7.4

### ISSUE THREAD

**Issue by hvr** [alpmestan/servant#42]
```
src/Servant/Response/Prelude.hs:12:8:
    Could not find module `GHC.Generics'
    It is a member of the hidden package `ghc-prim'.
    Perhaps you need to add `ghc-prim' to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
cabal: Error: some packages failed to install:
servant-response-0.1 failed during the building phase.
```

This is due to `GHC.Generics` living in `ghc-prim` for GHC 7.4 only; one way to address this is to use (like done in `deepseq.cabal`):

```
    -- GHC.Generics lived in `ghc-prim` for GHC 7.2 & GHC 7.4
    if impl(ghc < 7.6)
      build-depends: ghc-prim == 0.2.*
```

[I've edited the cabal file on Hackage to have a `base >= 4.6` constraint](https://hackage.haskell.org/package/servant-response-0.1/revisions/)

**Comment by alpmestan** [comment]
Oh, this package is deprecated and was a companion for servant 0.1 and some other packages. Don't waste your time with this one =)

**Comment by hvr** [comment]
Well... they do show up on http://matrix.hackage.haskell.org as requiring attention. :-)

I'll just add a few `servant < 0.2`  and `base` version constraints for those deprecated packages... that'll move them out of my sight for a long while then :-)

**Comment by alpmestan** [comment]
Yeah, that sounds like the best plan for this old friend of mine. Thank you very much for taking care of The Matrix =)
